src/components/AssetsResultsCount/index.jsx

```diff
@@ -27,7 +27,8 @@ const AssetsResultsCount = ({ paginationMetadata, filtersMetadata, searchMetadat
         values={{
           start: renderCount(paginationMetadata.start + 1),
           end: renderCount(paginationMetadata.end),
-          total: renderCount(paginationMetadata.totalCount),
+          formatted_total: renderCount(paginationMetadata.totalCount),
+          total: paginationMetadata.totalCount,
```

Comment:
This is the key - both are variables passed through to react-intl, but one is a number while the other is a span containing a number